### DISCUSSION TITLE
Ferramentas Front end

### DISCUSSION THREAD

**Discussion by Rahgomes** [frontendbr/forum#1420] [Dicas]
### Ferramentas Front end

Salve galera, bom feriado a todos e nada como um feriado de dia do trabalho "trabalhando"...
Enfim, piadas sem graça a parte rs venho aqui postar uma página sobre um compilado de ferramentas úteis para o dia-a-dia de um front end. 

Sei que há muitos sites com o mesmo intuito, muita gente guardando tais ferramentas nos favoritos algo que por sinal estava fazendo até ter essa iniciativa, consegui captar nessa breve jornada como front end muitas ferramentas relevantes e vi também que além de mim outros podem usufruir desses atalhos.

Link: http://ramongomes.com.br/ferramentasfrontend/

Se tiverem sugestões de outras ferramentas relevantes para enriquecermos ainda mais esse acervo com certeza serão bem vindas.

Essa é minha primeira postagem por aqui, mas com certeza pretendo ser mais participativo, pois vi que a comunidade é dahora todo mundo colaborando vamos para frente ( ͡° ͜ʖ ͡°)

**Comment by RenanSucs** [comment]
Boaa, vai ajudar bastante gnt! Parabéns, esse é o caminho.

**Comment by theandersonn** [comment]
Muito bom!
Parabéns pela iniciativa.

**Comment by pietrobelluno** [comment]
@Rahgomes Fala mano, apenas fazendo uma critica ao visual do site: acho que ta com muitos efeitos, principalmente quando passa o mouse em algum link, acho que ficaria melhor sem esse efeito, pra deixar mais clean, de resto muito boa sua iniciativa !

**Comment by mutheusalmeida** [comment]
O @pietrobp está certo. Muitos efeitos estragam a experiência do usuário e dão uma sensação de lentidão no site.

Ademais, parabéns pela iniciativa.

**Comment by Rahgomes** [comment]
@theandersonn @pietrobp @mutheusalmeida obrigado pelas críticas e elogios, realmente é muito bom ter outras opiniões, e tudo que for pra melhorar com certeza é bem vindo.

Tmj, sempre que for encontrando algo novo vou atualizando e se tiverem sugestões tamo ai!!!

**Comment by kivervinicius** [comment]
Torna o site mais clean não da nem pra abrir o site na empresa q o colega ja olha pro lado de tanto que chama atenção rs, os conteúdos estão muito bons!

**Comment by Rahgomes** [comment]
@kivervinicius demorou, na semana tava na correria mas esse final de semana acerto isso, além disso vou adicionar novas tools que encontrei e que vão ajudar tbm. valeuu

**Comment by tarciozemel** [comment]
Me lembrou bem o [https://recursosfrontend.desenvolvimentoparaweb.com/](https://recursosfrontend.desenvolvimentoparaweb.com/)...

Mas achei legal que complementou com mais recursos. 👍

**Comment by Rahgomes** [comment]
Sim @tarciozemel aliás esse site e outros foram os que me inspiraram pra fazer essa página, e a comunidade só tem a ganhar.

Valeuu tmj
